server/audit_math/supersimple.py

```diff
@@ -134,9 +133,9 @@ def compute_discrepancies(
 
         found = False
 
-        # Special case: if ballot can't be found by audit board, count it as a
-        # two-vote overstatement
-        if ballot_sample_cvr is None:
+        # Special cases: if ballot wasn't in CVR or ballot can't be found by
+        # audit board, count it as a two-vote overstatement
+        if ballot_sample_cvr is None or ballot_cvr is None:
```

Comment:
this is a math change, can we get a :+1: from @umbernhard ?

Reply:
So this can only possibly happen if we're sampling from the manifest, not the CVR. That might be another discussion like the one we just had @jonahkagan. I feel like we should be doing that...

In any case, I don't think this matters for now, since if we're just sampling from the CVR then the `or` will never be true.